stripper/ze_mapeadores_museum_v1.cfg

```diff
@@ -373,3 +372,71 @@ modify:
 		"OnPressed" "!self,CallScriptFunction,StopClaiming,,1"
 	}
 }
+
+// Delay nuke preperation to fix stages with longer than 10 second delays for zombie teleportation.
+modify:
+{
+	match:
+	{
+		"classname" "math_counter"
+		"targetname" "spaickbuttons"
+	}
+	delete:
+	{
+		"OnTrigger" "NumMapsPerRoundAdd125-1"
+	}
+	insert:
+	{
+		"OnTrigger" "NumMapsPerRound,Add,1,,,-1"
+	}
+}
```

Comment:
This seems invalid, `spaickbuttons` isn't a math_counter and this output only seems to exist in `MuseumPrepareNextStage`.

Reply:
My bad, I was testing something and forgot to remove it